models.py

```diff
@@ -1,11 +1,11 @@
 from flask_bcrypt import Bcrypt
 from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy.sql.elements import Null
```

Comment:
@iTaylorCodes how is removing this related to setting up sign up ?

Reply:
@alexkayabula Whoops, I removed it in an attempt to help Codacy coverage to pass